lib/shared/config-manager/src/index.ts

```diff
@@ -17,12 +21,18 @@ export class EnvironmentConfigManager {
     private readonly requestTimeoutMS: number
     private readonly cdnURI: string
     fetchConfigPromise: Promise<void>
-    private intervalTimeout?: NodeJS.Timeout
+    private intervalTimeout?: any
```

Comment:
why does this need to be any?

Reply:
basically because `NodeJS.Timeout` isn't supported in webworker / browser environments, they just return a number I believe. So the strategy here was to add a `setInterval` + `clearInterval` interface that would be passed in, so this needs to be an `Any` if we want it to compile when used in the non-NodeJS SDKs.

Reply:
if you wanted to get fancy you could use a type generic for this. Type it so that the generic is what is returned by the SetIntervalInterface and then use the same generic here.